lib/features/map_view/widgets/bottom_scroll_sheet/map_view_pop_behaviour.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../../controllers/bottom_sheet_controller.dart';
+import '../../controllers/controllers_set.dart';
+import '../map_config.dart';
+
+class MapViewPopBehaviour<T extends GoogleNavigable> extends ConsumerWidget {
+  const MapViewPopBehaviour({
+    required this.child,
+    required this.screenHeight,
+    super.key,
+  });
+  final double screenHeight;
+  final Widget child;
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final isAlmostFullyExtended =
```

Comment:
Is this needed here? I mean, could we extend the bottom sheet to let's say 50% of it's height?

Reply:
I added this cause the idea was to block view popping when the bottom sheet is fully extended - and reset the position of the sheet, instead of popping the view. And I've decided to give it a 2% margin of error (Idk for what reasons, just to be safe), but if you think we should remove this failsafe, I'm all for it

Reply:
Thanks for clarification. I thought the same but wanted to be sure. Actually it cost us nothing to have that safety future so in my opinion it is not wrong to have it :-)

Reply:
cool :)